internal/api/handlers.go

```diff
@@ -382,3 +434,17 @@ var GeneratePassword = func(length int) (string, error) {
 	}
 	return string(b), nil
 }
+
+// Helper function to generate a JWT
+func generateJWT(username, jwtSecret string) (string, error) {
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
+		"username": username,
```

Comment:
lets put the permissions here too. this will help us avoid going to the database every time we need to authorize someone.

Reply:
✅